assets/src/dashboard/app/api/apiProvider.js

```diff
@@ -62,12 +62,25 @@ export function reshapeStoryObject(editStoryURL) {
   };
 }
 
-export function reshapeTemplateObject({ id, title, pages }) {
+export function reshapeTemplateObject({
+  id,
+  title,
+  tags,
+  colors,
+  createdBy,
+  description,
+  pages,
```

Comment:
do any of these need defaults?

Reply:
For now... I don't think so.  All our templates are hardcoded so all the data is correct, and I imagine we can make sure when templates are created by users that we insert the appropriate default values too, but once we hook up the api, I'll definitely revisit.